packages/editor/src/client/routes/commentList.tsx

```diff
@@ -0,0 +1,408 @@
+            </DescriptionListItem>
+            <DescriptionListItem label={t('comments.panels.createdAt')}>
+              {currentComment?.createdAt && new Date(currentComment.createdAt).toDateString()}
+            </DescriptionListItem>
+            <DescriptionListItem label={t('comments.panels.updatedAt')}>
+              {currentComment?.modifiedAt && new Date(currentComment.modifiedAt).toDateString()}
+            </DescriptionListItem>
+            <DescriptionListItem label={t('comments.panels.revisions')}>
+              <Timeline align="left">
+                {currentComment?.revisions?.length
+                  ? currentComment?.revisions?.map(({text, createdAt}, i) => (
+                      <Timeline.Item key={i}>
+                        <div>{new Date(createdAt).toLocaleString()}</div>
+                        <RichTextBlock
+                          disabled
+                          // TODO: remove this
+                          onChange={console.log}
+                          value={text}
+                        />
+                      </Timeline.Item>
+                    ))
+                  : null}
+              </Timeline>
+            </DescriptionListItem>
+            {confirmAction === ConfirmAction.Reject ||
+            confirmAction === ConfirmAction.RequestChanges ? (
+              <DescriptionListItem
+                label={t(
+                  confirmAction === ConfirmAction.Reject
+                    ? 'comments.panels.rejectionReason'
+                    : 'comments.panels.requestChangesReason'
+                )}>
+                <Dropdown
+                  title={t(
+                    confirmAction === ConfirmAction.Reject
+                      ? 'comments.panels.rejectionReason'
+                      : 'comments.panels.requestChangesReason'
+                  )}
+                  placement="topStart">
+                  <Dropdown.Item
+                    key={CommentRejectionReason.Spam}
+                    active={CommentRejectionReason.Spam === rejectionReason}
+                    onSelect={() => setRejectionReason(CommentRejectionReason.Spam)}>
+                    {CommentRejectionReason.Spam}
+                  </Dropdown.Item>
+                  <Dropdown.Item
+                    key={CommentRejectionReason.Misconduct}
+                    active={CommentRejectionReason.Misconduct === rejectionReason}
+                    onSelect={() => setRejectionReason(CommentRejectionReason.Misconduct)}>
+                    {CommentRejectionReason.Misconduct}
+                  </Dropdown.Item>
+                </Dropdown>
+              </DescriptionListItem>
+            ) : null}
+          </DescriptionList>
+        </Modal.Body>
+        <Modal.Footer>
+          <Button
+            color={mapCommentActionToColor(confirmAction)}
+            disabled={isApproving || isRequestingChanges || isRejecting}
+            onClick={async () => {
```

Comment:
After confirming the action it should reload/refetch the data in the list. Refer to the `articleList` unpublish action for inspiration.

Reply:
I couldn't understand when the re-fetch happens in articleList route but I did it my way, please let me know your opinion